images/wavedrom/c-load_store_pair.adoc

```diff
@@ -0,0 +1,46 @@
+//## Zcmlsd instructions
+
+[wavedrom, ,svg]
+....
+{reg: [
+  {bits: 2, name: 'op',     type: 8, attr: ['2','C2']},
+  {bits: 5, name: 'imm',    type: 5, attr: ['5','offset[4:2|7:6]']},
+  {bits: 5, name: 'rd',     type: 5, attr: ['5','dest, dest[0]=0']},
+  {bits: 1, name: 'imm',    type: 1, attr: ['1','offset[5]']},
+  {bits: 3, name: 'funct3', type: 3, attr: ['3','C.LDSP']},
+], config: {bits: 16}}
+....
+
+[wavedrom, ,svg]
+....
+{reg: [
+  {bits: 2, name: 'op',     type: 8, attr: ['2','C2']},
+  {bits: 5, name: 'rs2',    type: 4, attr: ['5','src, src[0]=0']},
+  {bits: 6, name: 'imm',    type: 3, attr: ['6','offset[5:2|7:6]']},
+  {bits: 3, name: 'funct3', type: 8, attr: ['3','C.SDSP']},
+], config: {bits: 16}}
+....
+
+[wavedrom, ,svg]
+....
+{reg: [
+  {bits: 2, name: 'op',      type: 8, attr: ['2', 'C0']},
+  {bits: 3, name: 'rd`',     type: 3, attr: ['3', 'dest, dest[0]=0']},
+  {bits: 2, name: 'imm',     type: 2, attr: ['2', 'offset[2|6]']},
+  {bits: 3, name: 'rs1`',    type: 2, attr: ['3', 'base']},
+  {bits: 3, name: 'imm',     type: 3, attr: ['3', 'offset[5:3]']},
```

Comment:
Same here, offsets `[7:3]` instead of `[6:2]` - it seems you've copied the ones for `C.LW` instead of `C.LD`?

Reply:
Same applies to `C.SD`

Reply:
done thanks